getting-started/try-key-features-timescale-products.md

```diff
@@ -50,46 +50,41 @@ relational and time-series data from external files.
 
 <Procedure>
 
-1.  **Import some time-series data into your $HYPERTABLE**
+1.  **Import some time-series data into $HYPERTABLEs**
 
     1. Unzip <Tag type="download">[crypto_sample.zip](https://assets.timescale.com/docs/downloads/candlestick/crypto_sample.zip)</Tag> to a `<local folder>`.
 
-       This test dataset contains second-by-second trade data for the most-traded crypto-assets
-       and a regular table of asset symbols and company names.  
+       This test dataset contains second-by-second trade time-series data for the most-traded crypto-assets
```

Comment:
This will read better as two sentences. 1. This set contains ... 2. This data is best suited for ..., the other is for ...

Reply:
Nice point. Updated.